napari/layers/vectors/_vector_utils.py

```diff
@@ -69,7 +69,7 @@ def fix_data_vectors(
         if ndim does not match with third dimensions of vectors
     """
     if vectors is None:
-        vectors = []
+        vectors = np.array([])
```

Comment:
Is this necessary according to mypy? Seems weird given that `np.asarray` is called next.

Reply:
This is because `vectors` has type `Optional[np.ndarray]` (defined in the function signature), so assigning to a list is invalid.

Reply:
Oops - thanks for the clarification! Somehow I was only thinking about the return type.